app/src/main/java/com/github/onlynotesswent/model/notification/Notification.kt

```diff
@@ -8,7 +8,8 @@ data class Notification(
     val receiverId: String,
     val timestamp: Timestamp,
     val read: Boolean,
-    val type: NotificationType = NotificationType.DEFAULT
+    val type: NotificationType = NotificationType.DEFAULT,
+    val content: String? = null
```

Comment:
good job adding back content field for text messages. Did you put on max length on them?